janus_server/src/datastore.rs

```diff
@@ -140,22 +143,21 @@ impl Transaction<'_> {
         Ok(())
     }
 
-    /// Fetch the task parameters corresponing to the provided `task_id`.
-    //
-    // Only available in test configs for now, but will soon be used by
-    // aggregators to discover tasks from the database.
+    /// Construct a [`TaskParameters`] from the contents of the provided `Row`.
+    /// If `task_id` is `None`, it is used. Otherwise the task ID is read from
+    /// the row.
```

Comment:
```suggestion
    /// Construct a [`TaskParameters`] from the contents of the provided `Row`.
    /// If `task_id` is not `None`, it is used. Otherwise the task ID is read from
    /// the row.
```